tests/lib/eslint/eslint.js

```diff
@@ -335,6 +335,15 @@ describe("ESLint", () => {
 
                 processStub.restore();
             });
+
+            it("should throw an error if the flag 'unstable_ts_config' is used", () => {
```

Comment:
[question]: Is it more reasonable to regard it as a warning?

Reply:
Inactive flags would raise a warning before #18705 but now they throw errors because that makes it easier to track down problems. The drawback is that users will have to remove the flag for ESLint to keep working after the next update.